apps/site/assets/ts/stop/components/amenities/AccessibilityAmenityCard.tsx

```diff
@@ -1,15 +1,68 @@
 import React from "react";
-import AmenityCard from "./AmenityCard";
+import AmenityCard, { AmenityLink, AmenityModal } from "./AmenityCard";
 import { accessibleIcon } from "../../../helpers/icon";
+import { AccessibilityType } from "../../../__v3api";
+import Badge from "../../../components/Badge";
+import { accessibilityNames } from "../sidebar/Accessibility";
 
-const AccessibilityAmenityCard = (): JSX.Element => {
-  const icon = (
-    <span className="m-stop-page__icon">
-      {accessibleIcon("c-svg__icon-accessible-default")}
-    </span>
+const AccessibilityAmenityCard = ({
+  accessibleFeatures,
+  stopName
+}: {
+  accessibleFeatures: AccessibilityType[];
+  stopName: string;
+}): JSX.Element => {
+  const hasAccessibleFeatures = accessibleFeatures.includes("accessible");
+  const badge = !hasAccessibleFeatures && (
+    <Badge text="Not accessible" bgClass="u-bg--gray-lighter" />
+  );
+  return (
+    <AmenityCard
+      headerText="Accessibility"
+      icon={accessibleIcon("c-svg__icon-accessible-default")}
+      badge={badge}
+      modalContent={
+        hasAccessibleFeatures && (
+          <AmenityModal headerText={`Accessibility at ${stopName}`}>
+            <h2 className="h3">Station Features</h2>
```

Comment:
What is the reasoning behind using an `<h2>` tag with and `h3` class instead of just an `<h3>` tag?

Reply:
Just due to the structure of the content -- they're in a modal with `Accessibility at ${stopName}` in an h1 tag, so the next level down is h2. I'm just very used to following the levels sequentially, which helps convey a logical heading structure to assistive technologies.
Technically and even accessibility speaking, skipping heading levels isn't a huge violation, but some auditing tools such as [Lighthouse](https://developer.chrome.com/docs/lighthouse/overview/) will flag a warning for it.

Reply:
Okay, I think we might just have to agree to disagree.
Everything I find say that headers need to be in a hierarchical order, which I agree with.  The only thing I can find that say they need to be incrementally decreasing is the Lighthouse warning.  I think its perfectly fine to skip numbers of headers, just as long as they keep decreasing.  

I also feel like it makes reuse harder having to maintain a strict order if we have components with `<h2>` or `<h3>` tags and wish to reuse them on pages with different header structures.  IE the page we created has a component with an `<h2>` tag at the top.  Wish to reuse it on another page that already has an `<h2>` defined above it.

Reply:
I am also wondering whether you'd feel as strongly about this if the class names we were using here happened to be named differently. Like if we were working with `.top-heading`, `.sub-heading`, `.petite-heading`, and so on.